components/sysutils/ngrep/ngrep.p5m

```diff
@@ -13,6 +13,8 @@
 # Copyright 2016 Adam Stevko
 #
 
+<transform file path=usr.*/man/.+ -> default mangler.man.stability uncommitted>
```

Comment:
This is not strictly necessary, this transform is already present in transforms/defaults